internal/pkg/policy/monitor.go

```diff
@@ -225,8 +225,10 @@ func (m *monitorT) dispatchPending(ctx context.Context) {
 			m.log.Debug().
 				Str(logger.AgentID, s.agentID).
 				Str(logger.PolicyID, s.policyID).
-				Int64("rev", s.revIdx).
-				Int64("coord", s.coordIdx).
+				Int64("sub_rev", s.revIdx).
+				Int64("sub_coord", s.coordIdx).
+				Int64("rev", &policy.pp.Policy.RevisionIdx).
```

Comment:
we should make the log fields consistent, I've seen revision being logged out as `revision_idx` somewhere else
```
Policy coordinator added a new policy revision
revision_idx
```

Similarly, the action type if logged under different fields:
```
ack event
actionId

Action delivered to agent on checkin
id
```

Reply:
I think i got most of the revision/coordinator attributes to be consistent, but i've made a follow up to generally go through the codebase to check: https://github.com/elastic/fleet-server/issues/3239